plugins/algoltek-usb/fu-algoltek-usb-device.c

```diff
@@ -518,11 +520,15 @@ fu_algoltek_usb_device_write_firmware(FuDevice *device,
 
 	if (!fu_algoltek_usb_device_ers(self, 0x20, AG_IDENTIFICATION_128K_ADDR, error))
 		return FALSE;
-	if (!fu_algoltek_usb_device_ers(self, 0x20, AG_IDENTIFICATION_256K_ADDR, error))
-		return FALSE;
-	/* 1 sector = 4 kb, 256kb = 64 sector */
-	for (int i = 0; i < 64; i++) {
-		if (!fu_algoltek_usb_device_ers(self, 0x20, i, error))
+
+	if ((pid == 0x9421) || (pid == 0x9411)) {
+		/* 1 sector = 4 kb, 128kb = 32 sector */
+		for (int i = 1; i < 31; i++) {
+			if (!fu_algoltek_usb_device_ers(self, 0x20, i, error))
+				return FALSE;
+		}
```

Comment:
rather than hardcoding the PIDs in the .c file, I'd much rather define a flag we can use in the quirk file. I'm happy to convert the code -- but what should the flag name be? e.g. `ers-all-sectors` or `ers-128k` or something like that?

Reply:
It would be grateful.
The flag name can use `ers-all-sectors`.
I want to reconfirm, do you mean that except for PID=9421 or 9411 products,  other product can define the `ers-all-sectors`  flag?
PID=9421 or 9411 products will Skip first 4kb and then erase, other products will erase all sectors.

Reply:
Ohh, I meant a flag we can use for PID 0x9421 and 0x9411 -- is that perhaps `ers-not-first-sector` instead?

Reply:
Okay, the flag name can use `ers-not-first-sector`.

Reply:
Alternate proposal in https://github.com/fwupd/fwupd/pull/7960